src/Project/JobImportTF.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Job} from './Job';
+import {JobImport} from './JobImport';
+import {ToolArgs} from './ToolArgs';
+
+export class JobImportTF extends JobImport {
+  inputArrays?: string;
+  outputArrays?: string;
```

Comment:
Maybe?
```suggestion
  inputArrays: string;
  outputArrays: string;
```

Reply:
These two maybe undefined if not given, from the user.